src/tvshow/mod.rs

```diff
@@ -120,6 +120,7 @@ pub struct TVShow {
     pub last_episode_to_air: Option<EpisodeShort>,
     pub next_episode_to_air: Option<EpisodeShort>,
     pub networks: Vec<CompanyShort>,
+    #[serde(deserialize_with = "crate::util::default_on_null::deserialize")]
     pub number_of_episodes: u64,
```

Comment:
I've my doubt on this. I think I'd prefer making it optional to keep the idea that it was not specified

Reply:
Right since the tv show effectively have one episode, I didn't check that. A bit disappointing to complexify the data flow for such edge cases